src/core/index.ts

```diff
@@ -135,11 +137,21 @@ export class CrowdinValidationError extends CrowdinError {
 /**
  * @internal
  */
-function handleError<T>(error: any = {}): T {
-    if (Array.isArray(error.errors)) {
+function handleHttpClientError(error: unknown): never {
+    const crowdinResponseErrors =
+        error instanceof AxiosError
+            ? error.response?.data?.errors
+            : error instanceof FetchClientJsonPayloadError
+            ? error.jsonPayload && typeof error.jsonPayload === 'object' && 'errors' in error.jsonPayload
+                ? error.jsonPayload.errors
+                : null
+            : null;
```

Comment:
some devs don't like deep ternaries. i understand it, but find it the lesser of two evils when compared to nested mutations. using the ternaries generates a nice, clean typescript type and dodges the need for mutation.